frontend/awx/access/users/UserPage/UserTeams.tsx

```diff
@@ -33,7 +33,7 @@ export function UserTeams(props: { user: User }) {
   const selectTeamsAddUsers = useSelectTeamsAddUsers(view.selectItemsAndRefresh);
   const removeTeamsFromUsers = useRemoveTeamsFromUsers(view.unselectItemsAndRefresh);
   const { data } = useOptions<OptionsResponse<ActionsResponse>>('/api/v2/users/');
-  const canAddTeam = Boolean(data && data.actions && data.actions['POST']);
+  const canCreateUser = Boolean(data && data.actions && data.actions['POST']);
```

Comment:
should this `canAddUserToTeam`

Reply:
Sure - we use the presence of the POST attribute on an OPTIONS response to `/api/v2/users` to determine whether or not a user can add a team but also whether or not a user can add other users to a team.